### PR TITLE
BIC-200 # Home page quickly shown before redirecting to login page

### PR DESCRIPTION
**Pull request by GeetaSajwan**
- created middleware foe login
- created tests

### CODE REVIEW

File: src/bic/router/middleware/login.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+define(function (require) {
+  'use strict';
+
+  // local modules
+
+  var c = require('bic/console');
+  var API = require('bic/api');
+  var $ = require('jquery');
+  var _ = require('underscore');
```
Comment (jokeyrhyme) [inline src/bic/router/middleware/login.js]: I'm not sure this is worth holding up this PR, but...

`$` and `_` are not "local" modules, they're "foreign". So pedantically we should either remove the section comments, or rearrange the `require()`s so they are consistently arranged as they are in other modules with section comments.

**Comment by jokeyrhyme** [comment]
[Please, don’t commit commented out code](https://medium.com/@kentcdodds/please-don-t-commit-commented-out-code-53d0b5b26d5f)

File: tests/unit/router.js [hunk 2]
```diff
@@ -499,6 +499,22 @@ define([
...
+          done();
+        });
+      });
+
+      it('exports a Function, with 3 arguments', function () {
+        assert.isFunction(middleware);
+        assert.lengthOf(middleware, 3);
+      });
+    });
+
```
Comment (simonmarklar) [inline tests/unit/router.js]: is it possible/worth it to use sinon to stub out external functions and confirm the function works as intended?

**Comment by jokeyrhyme** [comment]
[Please, don’t commit commented out code](https://medium.com/@kentcdodds/please-don-t-commit-commented-out-code-53d0b5b26d5f)

**Comment by GeetaSajwan** [comment]
Did you mean that in tests?
I am writing one more test i will remove commented test in that commit.

File: tests/unit/router.js [hunk 3]
```diff
@@ -45,7 +47,8 @@ define([
             }
             return collection[id];
           },
-          set: function () { return null; }
+          set: function () { return null; },
+          findWhere:
```
Comment (jokeyrhyme) [inline tests/unit/router.js]: I believe this line is triggering a syntax error.

**Comment by jokeyrhyme** [comment]
LGTM :+1:

**Pull request merged**